### PR TITLE
Fixes Krav Maga allowing pacifism bypasses.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Fixes Krav Maga allowing pacifism bypasses.

## Why It's Good For The Game

Bugfixes are good for my GBP count. Also, pacifism bypasses bad.

## Changelog
:cl: CandleJaxx and Iamgoofball
fix: Fixes Krav Maga allowing pacifism bypasses.
/:cl:

### CODE REVIEW

**Review by Fikou — CHANGES_REQUESTED** [review]
your changelog needs to say what your pr does

**Comment by Iamgoofball** [comment]
@Fikou fixed it thanks sorry about that

**Comment by necromanceranne** [comment]
If this is a problem for krav maga, wouldn't this be an issue for all martial arts? Maybe this a more general check

**Comment by MrMelbert** [comment]
> If this is a problem for krav maga, wouldn't this be an issue for all martial arts? Maybe this a more general check

Pacifism is checked generically for `harm_act`, but no other `_act`s.

File: code/datums/martial/krav_maga.dm [hunk 1]
```diff
@@ -102,6 +102,8 @@
 /datum/martial_art/krav_maga/proc/leg_sweep(mob/living/attacker, mob/living/defender)
 	if(defender.stat != CONSCIOUS || defender.IsParalyzed())
 		return MARTIAL_ATTACK_INVALID
+	if(HAS_TRAIT(attacker, TRAIT_PACIFISM))
+		return MARTIAL_ATTACK_INVALID // Does 5 damage, so we can't let pacifists leg sweep.
```
Comment (LemonInTheDark) [inline code/datums/martial/krav_maga.dm:106]: I am become depression

Comment (Iamgoofball) [reply code/datums/martial/krav_maga.dm:106]: im gonna need u to elaborate

Comment (LemonInTheDark) [reply code/datums/martial/krav_maga.dm:106]: I hate that this is needed

**Pull request merged**